src/package/nuspec/Microsoft.CodeCoverage.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.CodeCoverage</id>
+    <version>$Version$</version>
+    <title>Microsoft.CodeCoverage</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Microsoft.CodeCoverage package brings infra for collecting code coverage from vstest.console.exe and "dotnet test".</description>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>vstest visual-studio unittest testplatform mstest microsoft test testing codecoverage code-coverage</tags>
+  </metadata>
+
+  <files>
+
+    <file src="Microsoft.CodeCoverage.props" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.VisualStudio.TraceDataCollector.dll" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\CodeCoverage\**\*.*"  exclude="**\*.pdb" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.VisualStudio.CodeCoverage.Shim.dll" target="lib\netstandard1.0\" />
```

Comment:
>CodeCoverage [](start = 61, length = 12)

Shim for Core/Full would need to be in separate .net frameworks (netcore1.0, net451)

Reply:
Done.